packages/taucmdr/cf/software/tau_installation.py

```diff
@@ -26,7 +26,6 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #
 """TAU software installation management.
-
```

Comment:
Please do not make style/whitespace changes as part of commits (and ideally PRs, but there is a time and a place for that) that change functionality. Style/whitespace changes should be in their own commit. In this instance it is also changing intentional spacing.

Please add back one newline here.